app/models/invitation.rb

```diff
@@ -2,4 +2,10 @@ class Invitation < ActiveRecord::Base
   belongs_to :board
   belongs_to :sender, :class_name => 'User'
   belongs_to :recipient, :class_name => 'User'
+
+  before_create :generate_token
+
+  def generate_token
+     self.token = SecureRandom.uuid
```

Comment:
no ma tyle znaków ze zawsze 'powinien' być uniq ale wyobraz sobie ze masz taki serwis jak facebook :) i nawet w FB były przypadki ze ktos przejoł sesje innej osoby :D a sesja ma jeszcze wiecej znaków ;) Chodzi mi o to byś się zastanowił na przyszłość jak takiemu czemuś zaradzić.